src/modules/invites/routes/invites.routes.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const router = express.Router();
+const { validate } = require('../../../middlewares/validate');
+const authService = require('../../auth/services/auth.service')
+const sharedService = require('../../../shared/services/response.services');
+const inviteSchema = require('../schemas/invite.schema');
+const inviteService = require('../services/invites.service')
+const userService = require('../../users/services/users.service');
+const { UserStatus } = require('../../../constants/constants');
+
+router.post('/accept-invite', inviteSchema.acceptInvite, validate, inviteService.acceptInviteUpdateUser, userService.updateSingleUser, authService.generateToken, authService.setTokenCookie, sharedService.successfullResponse)
+router.get('/resend-invite/:email', async (req, res, next) => {
+    const user = await userService.findOneByOptions({ email: req.params.email, status: UserStatus.PENDING })
```

Comment:
We can move this logic to invite service, then from invite service, other services will be called.

Reply:
We're using the invite service in user service while creating a user. Now if I use user service in the invite service there will be a circular dependency. If there is any way to avoid it, let me know so I will implement it.

Reply:
hmm, good problem. So what i can suggest is following

1. we can create seperate module like helper say user-helper, this may import invite service and perform invitations. So thus user module will not be dependent on invite module directly.

2. we can import service dynamically inside a function

Reply:
we can also do event emittions and receival between services

Reply:
For this i will suggest, with helper module or event triggering to invite module once user is created. and invite module on receiving event will trigger some functionality